tool/tctl/common/plugins_command.go

```diff
@@ -0,0 +1,106 @@
+/*
+ * Teleport
+ * Copyright (C) 2024  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package common
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/alecthomas/kingpin/v2"
+	"github.com/gravitational/trace"
+
+	pluginsv1 "github.com/gravitational/teleport/api/gen/proto/go/teleport/plugins/v1"
+	"github.com/gravitational/teleport/lib/auth"
+	"github.com/gravitational/teleport/lib/service/servicecfg"
+)
+
+// PluginsCommand allows for management of plugins.
+type PluginsCommand struct {
+	config     *servicecfg.Config
+	cleanupCmd *kingpin.CmdClause
+	pluginType string
+	dryRun     bool
+}
+
+// Initialize creates the plugins command and subcommands
+func (p *PluginsCommand) Initialize(app *kingpin.Application, config *servicecfg.Config) {
+	p.config = config
+
+	pluginsCommand := app.Command("plugins", "Manage Teleport plugins.").Hidden()
+	p.cleanupCmd = pluginsCommand.Command("cleanup", "Cleans up the given plugin type.")
+	p.cleanupCmd.Arg("type", "The type of plugin to cleanup.").StringVar(&p.pluginType)
+	p.cleanupCmd.Flag("dry-run", "Dry run the cleanup command.").BoolVar(&p.dryRun)
```

Comment:
This should default to `true`, and tell the user to run the command with `--no-dry-run` to actually perform the cleanup.